game/scripts/npc/abilities/leshrac_pulse_nova.txt

```diff
@@ -48,12 +48,6 @@
         "damage"                                          "100 180 260 500 740" //OAA
         "LinkedSpecialBonus"                              "special_bonus_unique_leshrac_3"
       }
-      "04" // unused
-      {
-        "var_type"                                        "FIELD_INTEGER"
-        "damage_scepter"                                  "160 190 220 450 680"
```

Comment:
why u delete this

Reply:
because Valve deleted it, it never worked anyway.